src/lib/template.js

```diff
@@ -10,20 +11,24 @@ let currentFields;
  *	@example
  *		template('foo{{bar}}', { bar:'baz' }) === 'foobaz'
  */
-export default function template(template, fields) {
-	currentFields = fields || EMPTY;
-	return template && template.replace(/\{\{([\w.-]+)\}\}/g, replacer);
+export default function template(template, fields, scope, dictionary) {
+	return template && template.replace(/\{\{([\w.-]+)\}\}/g, replacer.bind(null, fields || EMPTY, scope, dictionary));
```

Comment:
while this forces a `bind` to run on every instance, that isn't expensive and is much safer than using a file-scoped variable that changes on every instantiation